### PR TITLE
#94 Creating all the Alerts infrastructure Terraform 

### PR DESCRIPTION
**Pull request by bentrzcinski**
Created all the terraform needed to create all the infrastructure for the alerts system to be fully operational.
Also added variables files to all of the terraform folders so they can run within the folder.
Also ran all the terraform for the alerts system, so It should now be live on AWS.

### CODE REVIEW

File: terraform/rds/variables.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  description = "S3 database user name"
+  type        = string
+  sensitive   = true
+}
+
+variable "db_password" {
+  description = "password for the S3 database user"
+  type        = string
+  sensitive   = true
+}
```
Comment (ebradley12) [inline terraform/rds/variables.tf:11]: Love how you've only added relevant variables for each folder

**Review by ebradley12 — APPROVED** [review]
Perfecto

**Review by emily-curtis — APPROVED** [review]
looks good!

**Review by lukieh2014 — APPROVED** [review]
good work!

**Pull request merged**